lib/screens/alerts/parser.ex

```diff
@@ -1,6 +1,17 @@
 defmodule Screens.Alerts.Parser do
   @moduledoc false
 
+  def parse_result(%{"data" => data, "included" => included}) when is_list(data) do
+    facility_data = parse_facility_data(included)
+
+    alerts =
+      data
+      |> Enum.map(&parse_alert/1)
+      |> Enum.reject(&is_nil/1)
+
+    {alerts, facility_data}
```

Comment:
What do you think about using this parse function to put the facility data in with the informed entities? It seems like the `facility_data` map is basically used as a lookup table later for the facility ids in informed entities. It would be great to have that data put together here, so then we don't need to pass the facility lookup table all the way down to the widget instance. (But that means quite a bit of refactoring of the widget instance.) If this at all sounds like an interesting idea, please do get @jzimbel-mbta's weigh-in, too

Reply:
I liked that idea so I tried it out. Was a little tricky, but now we don't need the facility map anywhere outside of `parser.ex`.